engine/src/main/java/org/terasology/physics/shapes/SphereShape.java

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright 2017 MovingBlocks
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.terasology.physics.shapes;
+
+/**
+ * Represents a sphere shaped collision entity in the physics engine.
```

Comment:
See comment above. I guess another way you could write this is:

`* Represents a sphere shaped collision entity within the physics engine. Notice that the physics engine has its own entity concept/object, and this is not related to Terasology's entities.`

Reply:
Like you suggested above, renaming the `entity` references to `shape` (`Represents a box/sphere/convex hull/compound collision shape`) shall work in my opinion.